### ISSUE TITLE
[a11y] Cannot view list of users under collection

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1508]
![image](https://user-images.githubusercontent.com/37256050/167708592-09f33177-81b6-459f-84b4-35a4ff14f5de.png)

In the Security Panel I cannot load the list of users under each collection.

**Comment by maxxxxxdlp** [comment]
Duplicate of https://github.com/specify/specify7/issues/1433
